docs/CHANGELOG.rst

```diff
@@ -9,10 +9,13 @@ API Changes & Project structure changes
 - ``geotrellis-raster``
 
   - **Change:** ``geotrellis.raster.summary.polygonal.[Multi]TilePolygonalSummaryHandler`` replaced with ``geotrellis.raster.summary.polygonal.PolygonalSummary. Users should expect to implement concrete subclasses of ``geotrellis.raster.summary.GridVisitor`` and pass those to the new polygonalSummary methods. There are a number of default GridVisitor implementations provided for simple operations in `geotrellis.raster.summary.visitors``
+  - **New:** Created the ``geotrellis.raster.mapalgebra.focal.ZFactorCalculator`` and the ``geotrellis.raster.mapalgebra.focal.TileUnit`` types to improve
```

Comment:
Hm I think I should have mentioned it before, but mb it makes sense to add references to exact PRs with changes?